contrib/kind-common

```diff
@@ -605,6 +606,24 @@ spec:
     }
   ]
 }'
+EOF
+
+  cat <<EOF | oc apply -f -
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ovn-kubernetes
+  namespace: default
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "name": "ovn-kubernetes",
+    "type": "ovn-k8s-cni-overlay",
+    "ipam": {},
+    "dns": {},
+    "runtimeConfig": {}
+  }'
```

Comment:
@oshoval do you remember details about this ? 

I recall you had to provision this NAD to allow you to customize the default network attachment. 

Does it need to have this name ? i.e. `ovn-kubernetes` ? 

Are the other attributes mandatory ? i.e. `runtimeConfig` ?

Reply:
according notes, please check if this relevant
`docker exec ovn-worker cat /etc/cni/net.d/10-ovn-kubernetes.conf | jq`
seems i took from there

Reply:
thank you. 

This makes sense - you need a NAD place-holder to match the cluster default network configuration (which sits on the node's filesystem).

@kyrtapz / @qinqon we will need to find out if we can avoid having to provision this, and rely exclusively on the primary UDN NAD.

This is surely rely reaching out to NVIDIA, since they use this default network attachment customization feature.